server/src/schema/schema.js

```diff
@@ -0,0 +1,37 @@
+// Collection of mutations
+const {NewUserSignUp, UserLogin} = require('./mutation/User')
+// collection of queries
+const {GetAllUsersProfile, GetUserProfile} = require("../schema/query/query")
+
+const {
+  GraphQLObjectType,
+  GraphQLSchema,
+} = require('graphql');
+
+// RootQuery type
+const RootQuery = new GraphQLObjectType({
```

Comment:
could be named `queries` ? 
maintain consistent naming

Reply:
fixed this.